README.md

```diff
@@ -6,21 +6,33 @@ We created an online book store that allows a user to browse books by category o
 
 ## Table of Contents
 
+- [Installation](#installation)
 - [Usage](#usage)
 - [Technologies](#technologies)
 - [Credits](#credits)
 - [License](#license)
 
+## Installation
+
+First clone the repository from Github. After cloning you'll need to rename `.env.EXAMPLE` to `.env` and fill out the variables with your credentials, the three variables ending in `_KEY` are for emailjs which is for the contact form to send an email, those are optional. The `JWT_SECRET` must be filled out for your JWT. Now, in the project root folder,  we can run `npm run install` in our terminal, this will install all the necessary packages where its needed. 
 
 ## Usage
 
+After completing the installation, we can run `npm run develop` in our terminal. This should open a new window at localhost:3000.
+
 The user starts out at the home page. From there they can view featured books, browse by category, or use the search bar to look up a book. The user can also view their cart, login, or signup. Once logged in, they can place an order, edit their account details, or view their order history.
```

Comment:
We don't have featured books or the ability to browse by category. Also, the user can only view their cart if logged in and there is no order history to view.